### ISSUE TITLE
TBD: Introduce a database to the system

### ISSUE THREAD

**Issue by ecafer** [ecafer/film_website#2]
Can be relational database, or something else. Do a little research for what suits best in this scenario.

**Comment by ecafer** [comment]
I decided to go with PostgreSQL. Right now this is my understanding for the technologies I will use:

Flask app(Python) <-> psycopg2 <-> SQLAlchemy <-> PostgreSQL

**Comment by ecafer** [comment]
I decided to go with PostgreSQL. Right now this is my understanding for the technologies I will use:

Flask app(Python) <-> psycopg2 <-> SQLAlchemy <-> PostgreSQL

**Comment by ecafer** [comment]
I used PostgreSQL and psycopg2. I did not use SQLAlchemy. Thanks to psycopg2 I was able to connect to the database and execute SQL commands on the database. Right now, when the user submits the form consisting of the movie data, the data is POST' to the server, where it is passed to the database using psycopg2 and inserted in a table. I guess one needs to make sure that the db is up and running before starting the operation, besides that everything looks fine. I will test out, if the table still survives after I reboot the system.
